src/Dependencies/Microsoft.DiaSymReader/Version.targets

```diff
@@ -1,7 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <RoslynSemanticVersion>1.0.5</RoslynSemanticVersion>
-    <NuGetVersion>$(RoslynSemanticVersion)</NuGetVersion>
+    <NuGetVersion>$(RoslynSemanticVersion).1</NuGetVersion>
```

Comment:
...that's really not obvious. Especially when it's called "RoslynSemanticVersion".